### ISSUE TITLE
Prettier and eslint

### ISSUE THREAD

**Issue by samcolby** [BugiDev/react-native-calendar-strip#38]
I use [Prettier](https://github.com/prettier/prettier) on my all projects now, which has been fantastic and ensures a consistent formatting across the entire code base.

Combined with formatOnSave in VSCode, it ends up being a brilliant little syntax checker.  If the code doesn't 'move' when I click save, I know I'm missing a closing bracket or something similar.

Unfortunately, the included eslint rules in this project don't like the Prettier formatting.  Given the number of eslint errors in CalendarStrip.js anyway, can we change the eslint rules too.  I just use the following eslint rules now as I find the airbnb ones have become too strict.

```json
{
    "parser": "babel-eslint",
    "env": {
        "es6": true,
        "browser": true,
        "jest": true
    },
    "extends": [
        "eslint:recommended",
        "plugin:react/recommended"
    ],
    "plugins": [
        "react",
        "import"
    ],
    "rules": {
        "import/order": [
            "error"
        ],
        "no-unused-vars": [
            "error",
            {
                "args": "none"
            }
        ]
    }
}

```

**Comment by peacechen** [comment]
Looks good to me.  Thoughts, @BugiDev ?

**Comment by peacechen** [comment]
@samcolby Would you mind creating a PR?  The current .eslint file is quite extensive.  Should it be gutted and replaced with yours?

**Comment by samcolby** [comment]
Hey Peace,

I don't mind looking at, but it will be quite a large commit and wasn't sure if you would be better placed to do this.  It's fairly simple to set up [https://prettier.io/docs/en/usage.html](https://prettier.io/docs/en/usage.html)

> The current .eslint file is quite extensive. Should it be gutted and replaced with yours?

I think it should be gutted and replaced with one of the available presets. The above is just what I've settled on, but I'm happy to use which ever ones you're happiest with.  Maybe we should split this into 2 PRs anyway?  

Let me know what you think and if you want me to have a go at it.

Thanks
Sam

**Comment by peacechen** [comment]
I don't have experience with Prettier yet.  Typically I use Atom's jshint plugin for linting since projects vary widely.  I'll defer to you on what the best workflow is with Prettier.

**Comment by peacechen** [comment]
Merged PR #43 .  Thanks @samcolby
